docs/carbon-as-spark-datasource-guide.md

```diff
@@ -42,19 +42,20 @@ Now you can create Carbon table using Spark's datasource DDL syntax.
 
 | Property | Default Value | Description |
 |-----------|--------------|------------|
-| table_blocksize | 1024 | Size of blocks to write onto hdfs. For  more details, see [Table Block Size Configuration](./ddl-of-carbondata.md#table-block-size-configuration). |
+| table_blocksize | 1024 | Size of blocks to write onto hdfs. For more details, see [Table Block Size Configuration](./ddl-of-carbondata.md#table-block-size-configuration). |
```

Comment:
Can you avoid the unwated space changes, If not intentional?

Reply:
they have two space, we need fix it.

Reply:
OK